feat(CMS): add domain cms and phpdoc (#12)
Winzana/sherl-sdk-php: docs/docs/cms.md

| ------------- | -------------------------------------------------------------- | :----------------: | ---------------------- |
| **id**        | string                                                         | :white_check_mark: | ID of the media        |
| **uri**       | string                                                         | :white_check_mark: | URI of the media       |
| **width**     | number                                                         | :white_check_mark: | Width of the media     |
| **height**    | number                                                         | :white_check_mark: | Height of the media    |
| **caption**   | [CreateCaptionOutputDto](cms-types#CreateCaptionOutputDto)     | :white_check_mark: | Caption of the media   |
| **thumbnail** | [CreateThumbnailOutputDto](cms-types#CreateThumbnailOutputDto) | :white_check_mark: | Thumbnail of the media |

</details>

This call returns a [ArticleDto](cms-types#ArticleDto) object.

## Delete media

<span class="badge badge--warning">Require authentication</span>

```php
$sherlClient->cms->deleteMediaPage(string $id);
```

> This call returns a [ArticleDto](cms-types#ArticleDto) object.

Reviewer: ```suggestion
This call returns the deleted [ArticleDto](cms-types#ArticleDto) object.
```

## Get article by slug

<span class="badge badge--warning">Require authentication</span>

```php

> $cmsClient->getArticleBySlug(string $slug);

Reviewer 2: Revoir l'exemple de code pour plus de clarté :

```suggestion
$sherlClient->cms->getArticleBySlug(string $slug);
```

| **id**      | string | :white_check_mark: | ID of the FAQ      |
| **title**   | string | :white_check_mark: | Title of the FAQ   |
| **content** | array  | :white_check_mark: | Content of the FAQ |

</details>

This call returns a [ArticleDto](cms-types#ArticleDto) object.

## Create posts pages

<span class="badge badge--warning">Require authentication</span>

```php
$sherlClient->cms->createPostsPage(CMSArticleCreateInputDto $data);
```

<details>
<summary><b>CMSArticleCreateInputDto</b></summary>

| Fields        | Type   |      Required      | Description            |
| ------------- | ------ | :----------------: | ---------------------- |
| **id**        | string | :white_check_mark: | ID of the post         |
| **title**     | string | :white_check_mark: | Title of the post      |
| **content**   | string | :white_check_mark: | Content of the post    |
| **beginDate** | Date   | :white_check_mark: | Begin date of the post |
| **endDate**   | Date   | :white_check_mark: | End date of the post   |

</details>

This call returns a [ArticleDto](cms-types#ArticleDto) object.

## Find posts

<span class="badge badge--warning">Require authentication</span>

```php

> $sherlClient->cms->getPosts(FindPostsFiltersDto $filter);

Reviewer: ```suggestion
$sherlClient->cms->getPosts(FindPostsFiltersDto $filters);
```
